src/main/java/net/sf/jabref/gui/preftabs/AdvancedTab.java

```diff
@@ -61,6 +79,39 @@ public AdvancedTab(JabRefPreferences prefs) {
         useCaseKeeperOnSearch = new JCheckBox(Localization.lang("Add {} to specified title words on search to keep the correct case"));
         useUnitFormatterOnSearch = new JCheckBox(Localization.lang("Format units by adding non-breaking separators and keeping the correct case on search"));
 
+        defaultConsole = new JRadioButton(Localization.lang("Use default terminal emulator"));
```

Comment:
The preferences should be in "External programs", not in "Advanced", as "Console" is an external program.